modelservice/src/main/sql/build-db.sql

```diff
@@ -7,16 +7,18 @@ use aa_model_service;
 
 create table metric
 (
-  id        int unsigned primary key not null auto_increment,
-  m_key     varchar(100) unique not null,
-  hash      varchar(100) unique not null,
-  tags      json
+  id           int unsigned primary key not null auto_increment,
+  `key`        varchar(255) unique not null,
+  hash         char(36) unique not null,
+  description  varchar(255) unique not null,
+  tags         json
 );
 
 create table model_type
 (
-  id        smallint unsigned primary key not null auto_increment,
-  `key`     varchar(100) unique not null
+  id           smallint unsigned primary key not null auto_increment,
+  `key`        varchar(100) unique not null,
+  seyren_flag  boolean default false
```

Comment:
This belongs on the model table, not the model type table. It's individual models that we're trying to track as either coming from Seyren or not.